plasmapy/atomic/__init__.py

```diff
@@ -38,3 +38,11 @@
     nuclear_binding_energy,
     nuclear_reaction_energy,
 )
+
+proton = Particle("p+")
+electron = Particle("e-")
+neutron = Particle("n")
+positron = Particle("e+")
+deuteron = Particle("D 1+")
+triton = Particle("T 1+")
+alpha = Particle("He-4 2+")
```

Comment:
I'm wondering if there is a more efficient way of doing this. Maybe using a dictionary?
Might not be important if we are doing this just for a few particles.

Reply:
I like it how it is, given this is a shorthand for the most commonly used Pericles.

Oh, the wonders of autocorrect.